packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -153,6 +158,7 @@ export type InternalRequestParams = {
   headers?: Record<string, string> // TODO what is this
   transactionId?: number // TODO what is this
   unpacker?: Unpacker // TODO what is this
+  otelCtx?: Context // an otel context
```

Comment:
A saved instance of the context at the moment of creation of the prisma promise.